wesnoth-httt.po

```diff
@@ -578,7 +578,7 @@ msgstr "Sir Kaylan zemře"
 #. [label]
 #: data/campaigns/Heir_To_The_Throne/scenarios/02_Blackwater_Port.cfg:57
 msgid "Blackwater Port"
-msgstr "Přístav Temných vod"
+msgstr "Přístav Temná Voda"
```

Comment:
Nejsem si jist, jestli dobře rozumím vaší poznámce. Kde jsou často návěští velkými písmeny? Pokud tady v HttT, pak je to proto, že jsou většinou tvořena vlastními jmény. Nebo myslíte někde jinde?

Já si nahrál starší uloženou mapu S02 HttT a vidím, že [label] je text na pozici v mapě. Tedy pro mne jasně malým písmenem.

Ještě jsem prostudoval jednu papírovou cyklomapu, nad kterou jsem se ve svém přesvědčení utvrdil (bývalá výsypka, dočišťovací nádrž).
```suggestion
msgstr "přístav Temná Voda"
```

Reply:
Ano texty označující pozici na mapě jsou v kampaních s velkým písmenem na začátku. A většinou se rozhodně nejedná o názvy. Našel jsem to např. v SotA, tSG, DiD či tHoT.

Reply:
Dobrá, já to považuji za chybu, nebo spíše opomenutí překladatelů (včetně vlastního). Na mapě nenacházím pro velká písmena na začátku popisku žádné opodstatnění (nejsou to názvy kapitol). Zkrátka řeka je řeka a ne Řeka, hostinec je hostinec a ne Hostinec. A koukám - Public House není Veřejný dům ani veřejný dům.

Podle mého názoru se překládalo, aniž by někdo řešil, kde se to objeví a jestli je to správně. A pak se to tak nechalo. Dělat ale kvůli tomu z velkých písmen na začátku popisku na mapě dogma, to bych nerad.

Nechci to tlačit, jen bych to rád opravil, kde na to narazím (ve hře na mapě). Vidím, že [label] je i v popisu okna a tam zase bude mít pravděpodobně velké písmeno své místo. Takže nemám v plánu to hromadně měnit.

Reply:
Jasně, já jsem prostě vycházel z toho, co jsem viděl jinde. A zase mně se to s těmi velkými písmeny docela opticky líbí. 
Do budoucna se to tedy musí vyřešit. Zatím se to budu pokoušet to porůznu opravovat.